lib/content-services/src/lib/category/categories-management/categories-management.component.html

```diff
@@ -1,10 +1,20 @@
 <div class="adf-categories-management">
-    <p *ngIf="!categories.length && !categoryNameControlVisible"
-       class="adf-no-categories-message">
-        {{ noCategoriesMsg | translate }}
-    </p>
-    <div class="adf-categories-list"
-         [class.adf-categories-list-fixed]="!categoryNameControlVisible">
+    <div *ngIf="((!categoryNameControlVisible && categories.length)) || categoryNameControlVisible"
+         class="adf-category-name-field">
+        <mat-form-field  appearance="fill">
+            <input #categoryNameInput
+               matInput 
+               autocomplete="off"
+               [formControl]="categoryNameControl"
+               (keyup.enter)="addCategory()"
+               aria-labelledby="adf-category-name-input-label"
```

Comment:
this aria-labelledby seems incorrect because I can't find element with id adf-category-name-input-label so it is linked with nothing.